src/ildasm/dasm.cpp

```diff
@@ -339,6 +339,9 @@ BOOL Init()
 #ifdef FEATURE_CORECLR
 #ifdef FEATURE_PAL
     g_loader = CoreCLRLoader::Create(g_pszExeFile);
+    if (!g_loader) {
```

Comment:
Can you also add the similar check in ilasm side?
https://github.com/dotnet/coreclr/blob/master/src/ilasm/assem.cpp#L259